editor.planx.uk/src/pages/Team.tsx

```diff
@@ -26,72 +26,56 @@ import { client } from "../lib/graphql";
 import SimpleMenu from "../ui/SimpleMenu";
 import { useStore } from "./FlowEditor/lib/store";
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    backgroundColor: "#2C2C2C",
-    color: "#fff",
-    width: "100%",
-    flex: 1,
-    justifyContent: "flex-start",
-    // paddingTop: "calc(12vh + 75px)",
-    alignItems: "center",
-  },
-  back: {
-    textDecoration: "none",
-    color: "inherit",
-    display: "inline-flex",
-    alignItems: "center",
-    padding: theme.spacing(1, 2, 1, 1),
-    "&:hover": {
-      backgroundColor: "rgba(0,0,0,0.1)",
-    },
-  },
-  dashboard: {
-    backgroundColor: "#2C2C2C",
+const Root = styled(Box)(() => ({
+  backgroundColor: "#2C2C2C",
+  color: "#fff",
+  width: "100%",
+  flex: 1,
+  justifyContent: "flex-start",
+  alignItems: "center",
+}));
+
+const Dashboard = styled(Box)(({ theme }) => ({
+  backgroundColor: "#2C2C2C",
+  color: "#fff",
+  width: "100%",
+  maxWidth: 600,
+  margin: "auto",
+  padding: theme.spacing(8, 0, 4, 0),
+}));
+
+const DashboardList = styled("ul")(({ theme }) => ({
+  padding: theme.spacing(0, 0, 3),
+  borderBottom: "1px solid #fff",
+  margin: 0,
+}));
+
+const DashboardListItem = styled("li")(({ theme }) => ({
+  listStyle: "none",
+  position: "relative",
+  padding: theme.spacing(2.5, 2),
+}));
+
+const DashboardLink = styled(Link)(({ theme }) => ({
+  display: "block",
+  fontSize: theme.typography.h4.fontSize,
+  textDecoration: "none",
+  color: "currentColor",
+  fontWeight: 600,
+  marginBottom: theme.spacing(1.5),
+  marginTop: 0,
+}));
+
+const StyledSimpleMenu = styled(SimpleMenu)(({ theme }) => ({
+  position: "absolute",
+  top: theme.spacing(2),
+  right: theme.spacing(1),
+}));
+
+const LinkSubText = styled(Box)(() => ({
+  color: "#aaa",
+  "& a": {
     color: "#fff",
-    width: "100%",
-    maxWidth: 600,
-    margin: "auto",
-    padding: theme.spacing(8, 0, 4, 0),
-  },
-  dashboardList: {
-    padding: theme.spacing(0, 0, 3),
-    borderBottom: "1px solid #fff",
-    margin: 0,
-    "& li": {
-      listStyle: "none",
-    },
-  },
-  dashboardListItem: {
-    position: "relative",
-    padding: theme.spacing(2.5, 2),
-  },
-  dashboardLink: {
-    display: "block",
-    fontSize: theme.typography.h4.fontSize,
-    textDecoration: "none",
-    color: "currentColor",
-    fontWeight: 600,
-    marginBottom: theme.spacing(1.5),
-    marginTop: 0,
-  },
-  dashboardLinkWrap: {
-    // Add padding to allow text wrap before menu icon
-    paddingRight: theme.spacing(4),
-  },
-  menu: {
-    position: "absolute",
-    top: theme.spacing(2),
-    right: theme.spacing(1),
-  },
-  linkSubText: {
-    color: "#aaa",
-    "& a": {
-      color: "#fff",
-    },
-    "& + $linkSubText": {
-      marginTop: theme.spacing(1),
-    },
```

Comment:
I've removed this as it cannot currently happen - the `LinkSubText` component is never a sibling to another `LinkSubText`.